GOVERNANCE.md

```diff
@@ -9,6 +9,16 @@ It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2020
 The Technical Steering Committee is a group of Maintainers (Committers) who have earned the ability to modify (“commit”) sourcecode, documentation or other artifacts.
 This group is responsible for all technical oversight of StackStorm as Open Source Project.
 
+### TSC Meetings
+Because of the different schedules and time zones, TSC is keen to prioritize asynchronous collaboration and work.
+However, every `1st Tuesday` of the month `09:30 AM US Pacific`, [@StackStorm/tsc](OWNERS.md) members are gathering for `1 hour` to discuss the project problems, roadmap, priorities to help the StackStorm as a project evolve and succeed.
+The meetings are open, and we invite community to join them to be up-to-date with the latest project developments and progress.
+
+* TSC Meetings Host responsibility is rotated between the TSC maintainers.
+* Priority to host is given based on activity: Current Release Owners > Maintainers who weren't in rotation > Other Maintainer Volunteers.
+* Maintainers are free to decide collectively who would run the next TSC meeting. They can run in pairs, swap, however for diversification reasons no Maintainer can be a host for more than one in three consecutive meetings.
```

Comment:
I wasn't sure if this was a bit restrictive saying you can't be the host once for every 3 meetings, perhaps a better restriction is you can't be the same host for three or more consecutive meetings. If you are a release manager you might be the host for a couple of meetings if its nearing release time, and I think last time we had a meeting the volunteer for the host was the person who did it.
So maybe allowing 2 but not 3 consecutive would be a better restriction?

Reply:
Thanks :+1: was looking for feedback on this

Removed the limitation. Just stating the TSC Meeting rotation should be sufficient, hope everything else like the mechanism will follow naturally.